### ISSUE TITLE
NTR: Corpora amylacea

### ISSUE THREAD

**Issue by LCCarmody** [mgijax/mammalian-phenotype-ontology#4123]
Suggested term label:
Corpora amylacea

Definition (free text)
Corpora amylacea (CA) are polyglucosan bodies that accumulate residual substances originated from aging and both neurodegenerative and infectious processes.

Reference, in format PMID:#######

https://en.wikipedia.org/wiki/Corpora_amylacea. PMID: 34262556 PMID: 37358661

Parent term(s)
MP:0014239 abnormal intracellular organelle morphology (although I am not sure this is appropriate)

Children terms (if applicable) Should any existing terms that should be moved underneath this new proposed term?

Synonyms:
Exact: wasteosomes

These appear to be normal and increase in age, but may also be present at higher numbers in case of disease. They are formed in the cell, but seem to be released into the CSF, too.

@jmfostel

**Comment by LCCarmody** [comment]
oh, forgot to mention, we went to GO first, but they declined: https://github.com/geneontology/go-ontology/issues/30268

**Comment by sbello** [comment]
I was wondering about GO, good to know.

Additional references:
https://pubmed.ncbi.nlm.nih.gov/34634491/
https://pubmed.ncbi.nlm.nih.gov/10209236/ another way to write the term

Looking at GO definition of organelle "Organized structure of distinctive morphology and function. Includes the nucleus, mitochondria, plastids, vacuoles, vesicles, ribosomes and the cytoskeleton, and prokaryotic structures such as anammoxosomes and pirellulosomes. Excludes the plasma membrane."

Question is if it meets the distinctive morphology and function parts. I think function = waste container is okay but is the morphology distinctive and consistent enough? I'm leaning towards yes but will do some more digging. 

I think I would put this in as abnormal corpora amylacea morphology and possibly add the child term of abnormal corpora amylacea number as I'm guessing the most common phenotype is increased number.

Searching MGI I find one relevant phenotype from [25059425](https://pubmed.ncbi.nlm.nih.gov/25059425/) https://www.informatics.jax.org/allele/allgenoviews/MGI:3808349

**Comment by jmfostel** [comment]
our in-house definition is: corpora amylacea (shown in Renne et al. 2009), are small, laminated, basophilic bodies that are infrequently seen in the respiratory mucosa and nasal gland lumina of rats and mice.  our atlas only lists them in the nose.  the reference is: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1568333/ Renne R, Brix A, Harkema J, Kittel 

this paper includes "small agregates of mineralized debris in the mucosa" and "small basophilic bodies" both as descriptions of corpora amylacea.  there is a photomicrograph in the paper.  it could be similar to the one in Wikipedia, cannot be sure.  They are concretions, i think, just accumulating with either disease or age; not sure they would be considered to be waste containers.

having a child term of increased number would be great.  thank you for your work on this!

**Comment by jmfostel** [comment]
this is spot on!  thank you so much.

**Comment by sbello** [comment]
abnormal corpora amylacea number MP:0014674
increased corpora amylacea number MP:0014676
decreased corpora amylacea number MP:0014675

**Comment by jmfostel** [comment]
thank you!
